eya_def_tools/eya_def_tools/data_models/dataset.py

```diff
@@ -121,14 +119,16 @@ class DatasetDimension(StrEnum):
     WIND_SPEED = auto()
     WIND_FROM_DIRECTION = auto()
 
+    WIND_DATASET_ID = auto()  # Measurement station or reference dataset
+    POINT_ID = auto()  # Measurement point
+    HEIGHT = auto()  # Measurement height or turbine hub height
+
     WIND_FARM_ID = auto()
-    REFERENCE_WIND_FARM_ID = auto()
```

Comment:
I realised it does not make sense to separate between wind farm ID and reference wind farm ID. The same ID value can be used for both.